runtime/compiler/optimizer/EscapeAnalysis.cpp

```diff
@@ -2332,6 +2432,32 @@ bool TR_EscapeAnalysis::checkOverlappingLoopAllocation(TR::Node *node, TR::Node
    }
 
 
+
+void TR_EscapeAnalysis::collectAliasesOfAllocations(TR::Node *node, TR::Node *allocNode, bool collectAliases)
+   {
+   if (_visitedNodes->get(node->getGlobalIndex()))
+      return;
+
+   _visitedNodes->set(node->getGlobalIndex());
+
+   if (collectAliases && node->getOpCode().isStore() && node->getSymbol()->isAutoOrParm())
+      {
+      if (node->getFirstChild() == allocNode)
```

Comment:
code style - this if ladder should have {}s IMO just due to complexity. Given teh complexity of the conditions here can we put them one per line with the && at the star to the line followed by teh condition? This will reduce the line width and make this much more clear and easier to reason about. This is a style used elsewhere in the compiler when dealing with such complex conditions and has tended to work well.

Reply:
Cleaned this up in commit 414b74b10957a8ae1b8a4426f0c29dec2a905275